modyn/protos/evaluator.proto

```diff
@@ -54,25 +58,25 @@ message EvaluationStatusRequest { int32 evaluation_id = 1; }
 message EvaluationStatusResponse {
   bool valid = 1;
   bool is_running = 2;
-  bool state_available = 3;
-  bool blocked = 4;
-  optional string exception = 5;
```

Comment:
This is also problematic. We need to check for Exceptions at the evaluator and print them. I thought there was a problem with the visualization of the progress bars which I am fine to delete but I did not want to remove all status message transfer between the evaluator and the supervisor

Reply:
anyway, I will add back `batches_seen` and `samples_seen`, together with a new field `current_interval_idx`